_docs-2/troubleshooting/tools.md

```diff
@@ -59,6 +59,41 @@ Likewise, `rfile-info` will dump the key-value pairs and show you the contents o
     row columnFamily:columnQualifier [visibility] timestamp deleteFlag -> Value
     ...
 
+### Encrypted Files
+
+To examine an encrypted rfile the necessary encryption properties must be provided to the utility. To do this
+the `accumulo.properties` file can be copied, the necessary encryption parameters added, and then the properties file can 
+be passed to the utility with the `-p` argument. 
+
+For example, if using `PerTableCryptoFactory` and the `AESCryptoService`, you would need the following properties in
+your accumulo.properties file:
+
+    general.custom.crypto.key.uri=<path-to-key>/data-encryption.key
+    instance.crypto.opts.factory=org.apache.accumulo.core.spi.crypto.PerTableCryptoServiceFactory
+    table.crypto.opts.service=org.apache.accumulo.core.spi.crypto.AESCryptoService
+
+Example output:
+
+    $ accumulo rfile-info /accumulo/tables/1/default_tablet/A0000004.rf -p <path-to-properties>/accumulo.properties
+    2022-10-01T12:19:32,564 [rfile.PrintInfo] WARN : Attempting to find file across filesystems. Consider providing URI instead of path
```

Comment:
I think it's better to use the triple backtick method to create blocks, rather than rely on indentation, which is less obvious, and has more limitations (like no syntax highlighting).